packages/devextreme/testing/tests/DevExpress.core/utils.date.tests.js

```diff
@@ -107,6 +107,12 @@ QUnit.test('getIntervalByString second', function(assert) {
     assert.deepEqual(this.getDateIntervalByString('second'), { seconds: 1 });
 });
 
+QUnit.test('getMonthInterval should decrement month correctly (T1253076)', function(assert) {
+    const newDate = dateUtils.getMonthInterval(new Date(2025, 12, 0), -1);
+
+    assert.deepEqual(newDate, new Date(2025, 11, 0));
+});
+
```

Comment:
[test coverage]
Nikki, 1 test is definitely not enough for util))
If i replace your util's implementation with `new Date(2025, 11, 0)` the tests will be green